scripts/lib/aws/elasticache.sh

```diff
@@ -12,6 +12,29 @@ SCRIPTS_DIR="$ROOT_DIR/scripts"
 # functions for tests
 #################################################
 
+# k8s_controller_apply_temp_creds adds AWS temporary credentials
+# to service controller running on kubernetes cluster.
+# it depends upon:
+#   $service_name
+#   $AWS_ACCESS_KEY_ID
+#   $AWS_SECRET_ACCESS_KEY
+#   $AWS_SESSION_TOKEN
```

Comment:
Instead of relying on an environment variable called `$service_name`, please have the Bash function accept a parameter for service name. The other capitalized environment variables are fine to keep as-is.

Reply:
Updated scripts.